textext/asktext.py

```diff
@@ -438,9 +438,8 @@ def cb_word_wrap(self, widget=None, data=None):
         self._gui_config["word_wrap"] = self._word_wrap_tkval.get()
 
     def on_texcmd_change(self):
```

Comment:
The changes in this file seem to make these 2 functions redundant, should I just remove them?